gateway_test.go

```diff
@@ -650,7 +650,7 @@ func TestVersioningRequestFail(t *testing.T) {
 	recorder := httptest.NewRecorder()
 	req := testReq(t, "GET", "/", nil)
 	req.Header.Set("authorization", "zz1234")
-	req.Header.Set("version", "v1")
+	req.Header.Set("version", "v3")
```

Comment:
Why you set it to v3?

Reply:
I can't actually remember at this point. Will check when retesting after changes.